linter/rules/primary_key_is_not_first_dimension_in_view.py

```diff
@@ -0,0 +1,17 @@
+from linter.rule import Rule
+from typing import Any, Tuple
+
+
+class PrimaryKeyIsNotFirstDimensionInView(Rule):
+    def applies_to() -> Tuple[str, ...]:
+        return ('view',)
+
+    def run(self, view: Any) -> bool:
+        dimensions = view.get('dimensions', [])
+        primary_key_is_first = True
+        if len(dimensions) > 0:
```

Comment:
This can be shortened to:

```
dimensions = view.get('dimensions', [])
if len(dimensions) > 0 and 'primary_key' not in dimensions[0]:
    return False
return True
```